model_cards/gpt2-README.md

```diff
@@ -79,7 +80,19 @@ output = model(encoded_input)
 ### Limitations and bias
 
 The training data used for this model has not been released as a dataset one can browse. We know it contains a lot of
-unfiltered from the internet, which is far from neutral. Therefore, the model can have biased predictions:
+unfiltered from the internet, which is far from neutral. As the openAI team themselves point out in their
```

Comment:
unfiltered content? data?

Reply:
Oh yes, content is fine. Thanks for flagging!